examples/query_lang/credentials_alternatives.json

```diff
@@ -54,18 +54,27 @@
         "vct_values": ["https://company.example/company_rewards"]
       },
       "claims": [
-        {"id": "current", "path": ["rewards_number"]},
-        {"id": "legacy", "path": ["legacy_system_rewards_number"]}
+        {"id": "current", "path": ["rewards_number"]}
       ],
       "claim_sets": [
-        ["current", "legacy?"]
+        ["current"]
```

Comment:
As long as claim_sets is preserved, the `?` semantic is still here, but hidden. Either we remove claim_sets or we should keep `?` as an option.